src/app/components/ContentWrapper.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+// import 'terra-base/lib/baseStyles';
+
+import styles from './ContentWrapper.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The content to be placed within the main content area of the container.
+   */
+  content: PropTypes.func,
+
+  props: PropTypes.object,
+};
+
+const defaultProps = {
+  content: undefined,
+  props: undefined,
+};
+
+const ContentWrapper = ({ content: Content, props }) => {
+  const contentClassNames = cx([
+    'site-content',
+  ]);
+
+  return (
+    <div
+      data-terra-dev-site-content
+      className={contentClassNames}
```

Comment:
this could be simplified to be `className={cx('site-content')}` since we have one class name